en/gps_compass/septentrio_asterx-rib.md

```diff
@@ -0,0 +1,209 @@
+Weight = 50g
+
+<div style="text-align: center;">
+
+![](../../assets/hardware/gps/septentrio_sbf/AsteRX-m3+RIB-board.png "Septentrio Robotics Interface Board")
+
+</div>
+
+## Table Of Contents
+1. [Key Features](../../en/gps_compass/septentrio_asterx-rib.md#key-features)
+2. [Purchase](../../en/gps_compass/septentrio_asterx-rib.md#purchase)
+3. [Interfaces](../../en/gps_compass/septentrio_asterx-rib.md#interfaces)
+   1. [USB](../../en/gps_compass/septentrio_asterx-rib.md#usb)
+   2. [44-pin Header](../../en/gps_compass/septentrio_asterx-rib.md#44-pin-header)
+   3. [LED's](../../en/gps_compass/septentrio_asterx-rib.md#leds)
+   4. [LOG Button Header](../../en/gps_compass/septentrio_asterx-rib.md#log-button-header)
+   5. [PPS/Event Header](../../en/gps_compass/septentrio_asterx-rib.md#ppsevent-header)
+   6. [Power Supply Options](../../en/gps_compass/septentrio_asterx-rib.md#power-supply-options)
+4. [Hardware Setup](../../en/gps_compass/septentrio_asterx-rib.md#hardware-setup)
+   1. [Dual Antenna](../../en/gps_compass/septentrio_asterx-rib.md#hardware-setup)
+   2. [Web App](../../en/gps_compass/septentrio_asterx-rib.md#web-app)
+5. [PX4 Setup](../../en/gps_compass/septentrio_asterx-rib.md#px4-setup)
+   1. [Single Antenna](../../en/gps_compass/septentrio_asterx-rib.md#single-antenna)
+   2. [Dual Antenna](../../en/gps_compass/septentrio_asterx-rib.md#px4-setup)
+
+## Key Features
+
+- Credit-card size boards with low power consumption
+- Easy-to-integrate into any system
+- Best-in-class RTK performance with true multi-constellation, multi-frequency GNSS technology
+- Advanced Interference Mitigation (AIM+) anti-jamming and anti-spoofing technology
+- Resilient to vibrations and shocks
+- 44 pins I/O connector for autopilots such as Pixhawk
+- On-board logging
+- Micro USB connector
+
+## Purchase
+
+All AsteRx receivers and Robotic Interface Boards can be purchased from the Septentrio webshop:
+
+- [AsteRx-m3 Pro](https://www.septentrio.com/en/products/gnss-receivers/oem-receiver-boards/asterx-m3-pro)
+- [AsteRx-m3 Pro+](https://www.septentrio.com/en/products/gnss-receivers/oem-receiver-boards/asterx-m3-pro-plus)
+
+Other PX4 supported devices from Septentrio:
+- [mosaic-go evaluation kit](../../en/gps_compass/septentrio_mosaic-go.md)
+
+## Interfaces
+
+<div style="text-align: center;">
+
+![Septentrio Robotics Interface Board Fritzing drawing](../../assets/hardware/gps/septentrio_sbf/RIB.png "Septentrio Robotics Interface Board Fritzing drawing")
+
+</div>
+
+### USB
+_Connector type: micro USB type B._
+
+The micro USB type B connector can be attached to a PC to power the receiver and to communicate with it over its USB
+port.
+
+### 44-pin header
+_Connector type: SAMTEC TMM-122-03-S-D, 2-mm pitch._
+
+The 44-pin header can be used to connect multiple GPIO devices. Please refer to
+the [hardware manual](https://www.septentrio.com/system/files/support/asterx-m3_product_group_hardware_manual_2.2.0_1.pdf)
+for the pinout.
+
+### LED's

```

Comment:
PX4 typically emits vehicle ready-to-fly information on GPS LEDs - is that setup supported? https://docs.px4.io/master/en/getting_started/led_meanings.html#ui-led

Reply:
Unfortunately, this is not supported, as an additional microcontroller is required to drive the LEDs.

I will submit a request to Septentrio to be able to support this though, as it is a useful feature.